### ISSUE TITLE
Support Timber Vector agent, add node.js support

### ISSUE THREAD

**Issue by anutator** [opensearch-project/OpenSearch#720]
**Is your feature request related to a problem? Please describe.**
I found that you give examples in OpenSearch Dashbord for filebeat and metricsbeat.
Can you add support to Timber Vector https://vector.dev/ and use it instead of filebeat. They assert that Vector is faster than 
Filebeat (up to 10x), FluentBit, FluentD, Logstash, SplunkHF, SplunkUF.
Is metricbeat as fast as Vector? Does it collect only metrics and not logs?

Please add support for Node.js logs (pm2) and metrics to OpenSearch dashboard.

**Comment by jkowall** [comment]
Vector is already supported. We currently have the same APIs as elasticsearch.￼

**Comment by stockholmux** [comment]
@jkowall You're right. It should work. I'd love to see this confirmed. I know @ryn9 has done quite a bit of work with this on Open Distro.

**Comment by ryn9** [comment]
Regarding Vector-

Opendistro and the ASWS hosted solution are supported in Vector.
Have a look here - https://vector.dev/docs/reference/configuration/sinks/elasticsearch/.

One feature that these tools largely miss out on though - other than Logstash - is DLQ.
Read about that here: https://www.elastic.co/guide/en/logstash/current/dead-letter-queues.html

As such - you may want to do something like distributed Vector's to a centralized Logstash - which in turn handles the actual indexing.

**Comment by dblock** [comment]
I'm moving this to OpenSearch Dashboards. I think the ask is to provide documented examples of how to use these other agents.
